packages/teleport-shared/package.json

```diff
@@ -28,7 +28,7 @@
   "dependencies": {
     "@babel/types": "^7.5.5",
     "@teleporthq/teleport-types": "^0.10.0-alpha.3",
-    "jss": "^9.8.7",
+    "jss": "^10.0.0",
```

Comment:
can you also check the latest version of "jss-preset-default"?

Reply:
Updated [here](https://github.com/teleporthq/teleport-code-generators/pull/381/commits/de4d58127c5cef1cc0ff3e1bff32a312ef967240#diff-474984405b58fbdd16370798f34ff9e6R32).